tests/nlp/checks/model_evaluation/prediction_drift_test.py

```diff
@@ -95,3 +95,33 @@ def test_just_dance_small_drift(just_dance_train_test_textdata_sampled):
     ))
 
     assert_that(result.value['Drift score'], close_to(0.05, 0.01))
+
+
+def test_token_classification(small_wikiann_train_test_text_data):
+    # Arrange
+    train, test = small_wikiann_train_test_text_data
+    check = PredictionDrift()
+
+    # Act
+    result = check.run(train, test, train_predictions=np.asarray(train.label),
+                       test_predictions=np.asarray(test.label))
+
+    # Assert
+    assert_that(result.value['Drift score'], close_to(0, 0.01))
+
+
+def test_token_classification_with_nones(small_wikiann_train_test_text_data):
+    # Arrange
+    train, test = small_wikiann_train_test_text_data
+    train_label = train.label
+    train_label[0][0] = None
```

Comment:
you got confused in the tests

Reply:
Nope, didn't :) note that I take the label and pass it as the prediction as I have no model predictions for this test dataset